homeassistant/components/squeezebox/media_player.py

```diff
@@ -587,4 +590,23 @@ async def async_browse_media(self, media_content_type=None, media_content_id=Non
             "search_id": media_content_id,
         }
 
-        return await build_item_response(self._player, payload)
+        return await build_item_response(self, payload)
+
+    async def async_get_browse_image(self, browse_image):
+        """Get album art from Squeezebox server."""
+        internal_url = f"{self._internal_artwork_url}{browse_image}"
+        result = await self._async_fetch_image(internal_url)
+        if result == (None, None):
+            _LOGGER.info("Error retrieving proxied album art from %s", internal_url)
+        return result
+
+    async def async_browse(self, *args, **kwargs):
+        """Browse player's media library."""
+        return await self._player.async_browse(*args, **kwargs)
+
+    def get_thumbnail_url(self, image_url):
```

Comment:
This method makes no sense, it's supposed to fetch it from the squeezebox server right ?

Reply:
Not needed anymore and removed. Thanks for working on this one @ctalkington.